CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ cmake_minimum_required(VERSION 3.8)
 
 project(cmcstl2 CXX)
 
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
This should *not* be a property of the exported `stl2` interface library: we don't want to mandate whether or not users can use compiler extensions. Ideally it would be a property of an internal-only interface library to which all the examples/tests link.

Reply:
Huh, I didn't realise these propagate to users as well.

Reply:
[Looking at this thread][1], I don't know if it's possible to achieve what you're hoping for, at least in CMake 3.8. An alternative would be to change line 37 to `target_compile_options(stl2 INTERFACE -std=c++17)`.

[1]

Reply:
Some investigation demonstrates that the value of `CXX_EXTENSIONS` doesn't propagate into the properties of the installed interface library. Which I suppose is only logical, since there's no way to set the property on an interface library.